frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -84,6 +84,15 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
       },
       replicas: 1,
     },
+    serverless: {
+      enabled: false,
+      scaling: {
+        minpods: 1,
+        maxpods: 1,
+        concurrencyrequest: 1,
+        concurrencylimit: 1,
```

Comment:
Did you confirm these are appropriate defaults?

Reply:
Shouldn't these be optional?

Reply:
I'll confirm the default values, for now have kept it `''` and if user doesn't change it shouldn't add above annotations in payload as below while creating knative resources

```
apiVersion: serving.knative.dev/v1alpha1
kind: Service
metadata:
  name: autoscale-go
  namespace: default
spec:
  template:
    metadata:
      annotations:
        # Knative concurrency-based autoscaling (default).
        autoscaling.knative.dev/class: kpa.autoscaling.knative.dev
        autoscaling.knative.dev/metric: concurrency
        # Target 10 requests in-flight per pod.
        autoscaling.knative.dev/target: "10"
        # Disable scale to zero with a minScale of 1.
        autoscaling.knative.dev/minScale: "1"
        # Limit scaling to 100 pods.
        autoscaling.knative.dev/maxScale: "100"
    spec:
      containers:
        - image: gcr.io/knative-samples/autoscale-go:0.1
```

Details can be seen here https://github.com/knative/docs/tree/master/docs/serving/samples/autoscale-go